models/issues/review_test.go

```diff
@@ -162,7 +162,7 @@ func TestGetReviewersByIssueID(t *testing.T) {
 		},
 	)
 
-	allReviews, err := issues_model.GetReviewsByIssueID(db.DefaultContext, issue.ID)
+	allReviews, _, err := issues_model.GetReviewsByIssueID(db.DefaultContext, issue.ID)
```

Comment:
Test should cover the return values

Reply:
a893d27ec7b86bf25d8822a85aa0cf5b0e7a4d94